application/main.py

```diff
@@ -0,0 +1,80 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+from application.database import SessionLocal, engine, Base
+from application import models, schemas, crud, temperature
+
+
+Base.metadata.create_all(bind=engine)
+
+app = FastAPI()
+
+
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.post("/cities", response_model=schemas.City)
+def create_city(city: schemas.CityCreate, db: Session = Depends(get_db)):
+    return crud.create_city(db=db, city=city)
+
+
+@app.get("/cities", response_model=list[schemas.City])
+def get_cities(skip: int = 0, limit: int = 10, db: Session = Depends(get_db)):
+    return crud.get_cities(db=db, skip=skip, limit=limit)
+
+
+@app.get("/cities/{city_id}", response_model=schemas.City)
+def get_city(city_id: int, db: Session = Depends(get_db)):
+    db_city = crud.get_city(db=db, city_id=city_id)
+    if db_city is None:
+        raise HTTPException(status_code=404, detail="City not found")
+    return db_city
+
+
+def update_city(db: Session, city_id: int, city: schemas.CityCreate):
+    db_city = db.query(models.City).filter(models.City.id == city_id).first()
+    if db_city is None:
+        return None
+    db_city.name = city.name
+    db_city.description = city.description
```

Comment:
The `update_city` function attempts to update a `description` attribute, but the `schemas.CityCreate` schema does not include this field. Ensure that the schema matches the attributes being updated to avoid errors.